pipelines/forecast_pyrenew.py

```diff
@@ -179,6 +181,57 @@ def create_hubverse_table(model_fit_path):
     return None
 
 
+def get_pmfs(param_estimates: pl.LazyFrame, loc_abb: str, disease: str):
```

Comment:
See also `filter_params` here: https://github.com/cdcent/stf-pyrenew-added-value/blob/main/src/stf_pyrenew_added_value/pull_data_as_of.py#L75

I would rather import this functionality from `pyrenew-hew`, so please take a look at that code and make sure this update is the best of both worlds.